mne/utils/docs.py

```diff
@@ -3130,9 +3130,9 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     suppression.
     When ``method='iir'``, ``phase='zero'`` (default) or
     ``phase='zero-double'`` constructs and applies IIR filter twice, once
-    forward, and once backward (making it non-causal) using filtfilt.
+    forward, and once backward (making it non-causal) using `~scipy.signal.filtfilt`.
```

Comment:
without the `:func:` prefix, the parentheses aren't rendered.

```suggestion
    forward, and once backward (making it non-causal) using
    :func:`~scipy.signal.filtfilt`.
```